apps/api/src/server/app/validation/validation.test.js

```diff
@@ -19,3 +19,19 @@ test('gets the appeal information', async (t) => {
 	t.is(resp.status, 200);
 	t.deepEqual(resp.body, [validationLineNew, validationLineIncomplete]);
 });
+
+test('gets the appeallant information', async (t) => {
+	const resp = await request.get('/validation/:id');
+	const appealReviewInfo = {
+		AppealId : 1,
+		AppealReference: 'APP/Q9999/D/21/1345264',
+		AppellantName: 'Lee Thornton',
+		AppealStatus:'new',
+		Received: '23 Feb 2022',
+		AppealSite:'96 The Avenue, Maidstone, Kent, MD21 5XY',
+		LocalPlanningDepartment: 'Maindstone Borough Council',
+		PlanningApplicationReference: '48269/APP/2021/1482'
+	};
+	t.is(resp.status, 200);
+	t.deepEqual(resp.body, [appealReviewInfo]);
```

Comment:
So once we remove the square braces in `validation.controller.js`, then we will need to change this line to `t.deepEqual(resp.body, appealReviewInfo)`